modules/documents/client/views/document-manager.html

```diff
@@ -278,8 +278,9 @@
 								</span>
 								<span hidden class="col author-col" title="{{ doc.documentAuthor }}">
 									{{ doc.documentAuthor }}
-								</span>
-								<span class="col date-added-col">{{ doc.documentDate | documentDateFilter: doc.documentDateDisplayMnYr }}</span>
+                </span>
+                <span ng-if="doc.inspectionReport.dateResponse" class="col date-added-col">{{ doc.inspectionReport.dateResponse | documentDateFilter: doc.documentDateDisplayMnYr }}</span>
```

Comment:
Change seems fine to me. For the sake of personal preference (readability too maybe?), would a ternary operator work here? I am thinking something like:
```
<span class="col date-added-col">
{{ doc.inspectionReport.dateResponse ? doc.inspectionReport.dateResponse :  doc.documentDate | documentDateFilter: doc.documentDateDisplayMnYr }}
</span>
```